include/clap/ext/thread-pool.h

```diff
@@ -57,7 +57,7 @@ typedef struct clap_host_thread_pool {
    // Returns true if the host did execute all the tasks, false if it rejected the request.
    // The host should check that the plugin is within the process call, and if not, reject the exec
    // request.
-   // [audio-thread]
+   // [audio-thread & active & processing]
    bool(CLAP_ABI *request_exec)(const clap_host_t *host, uint32_t num_tasks);
```

Comment:
I think audio-thread already implies active and processing, but it doesn't hurt to repeat.